beerlog/gui/achievements_tests.py

```diff
@@ -0,0 +1,108 @@
+"""Tests for display.py"""
+
+import multiprocessing
+import os
+import tempfile
+import unittest
+
+import PIL
+
+from beerlog import beerlogdb
+from beerlog.gui import achievements
+from beerlog.gui import display as beerlog_display
+
+# pylint: disable=protected-access
+
+class FakeLumaDevice():
+  """Fake LumaDevice to provide proper sizes"""
+  def __init__(self):
+    self.width = 128
+    self.height = 64
+    self.size = (self.width, self.height)
+
+
+class AchievementsTests(unittest.TestCase):
+  """Tests for the Achivements generation"""
+
+  DB_PATH = ':memory:'
+
+  def CheckAgainstGolden(self, display, achievement, golden_path):
+    """Makes sure the image matches"""
+    with tempfile.NamedTemporaryFile(suffix='.jpg') as temp:
+      display._ShowAchievement(achievement, picture=temp.name)
+      try:
+        self.assertTrue(self.ComparePictures(golden_path, temp.name), (
+            'Result picture for {0:s} doesn\'t match "{1:s}"\n'
+            'You can generate new golden images using tools/generate_golden.py'
+            ).format(achievement.__class__.__name__, golden_path))
+      except AssertionError as e:
+        (out_f, out_p) = tempfile.mkstemp(suffix='.jpg')
```

Comment:
Could not `temp.name` be copied using - for example `shutil.copy` - instead of using `mkstemp` ?

Reply:
I don't have a temp image file now

Reply:
Isn't `temp.name` an available temp image file ?